eng/build.yml

```diff
@@ -412,8 +411,11 @@ jobs:
           displayName: Run dotnet new Integration Tests
 
       - ${{ if not(contains(parameters.agentOs, 'Windows_NT')) }}:
-        - script: eng/common/cibuild.sh
+        - script: eng/common/build.sh
                     --configuration $(_BuildConfig)
+                    --restore
```

Comment:
will template packages still be published with this change?

Reply:
Good point! Template packages should be published as before.
So far as I know packing and publishing packages are done by original Windows_NT job. The change was just made to our separate template engine job. It should not affect Windows_NT job.